main.tf

```diff
@@ -37,15 +37,23 @@ module "stacks" {
   worker_pool_id = var.worker_pool_id
   runner_image   = var.runner_image
 
-  push_policy_id    = spacelift_policy.push.id
-  plan_policy_id    = spacelift_policy.plan.id
-  trigger_policy_id = spacelift_policy.trigger_dependency.id
+  access_policy_id  = var.access_policy_id_override != null ? var.access_policy_id_override : spacelift_policy.access.id
+  push_policy_id    = var.push_policy_id_override != null ? var.push_policy_id_override : spacelift_policy.push.id
+  plan_policy_id    = var.plan_policy_id_override != null ? var.plan_policy_id_override : spacelift_policy.plan.id
```

Comment:
```suggestion
  plan_policy_id    = var.plan_policy_id != null ? var.plan_policy_id : spacelift_policy.plan.id
```

Reply:
also update all policies to not create them if we provide IDs of existing policies

```
resource "spacelift_policy" "plan" {  
  count = var.plan_policy_id == null ? 1 : 0
  
  type = "PLAN"  
  name = "Global Plan Policy"  
  body = file("${path.module}/policies/plan-global.rego")
}
```